WordPress/src/main/java/org/wordpress/android/ui/posts/prepublishing/home/compose/PrepublishingHomeSocialItem.kt

```diff
@@ -168,7 +183,7 @@ fun PrepublishingHomeSocialItemPreview() {
                     TrainOfIconsModel(R.drawable.ic_social_instagram),
                     TrainOfIconsModel(R.drawable.ic_social_tumblr),
                 ),
-                modifier = Modifier.fillMaxWidth()
+                modifier = Modifier.fillMaxWidth().padding(16.dp)
```

Comment:
💡 Would it make sense to use `Margin.ExtraLarge` instead of `16.dp` for the padding?

Reply:
Since this is just a preview I don't think there's much value in using the constant here.

I think it could actually be bad because using it here would make it show up in the list of usages of that constant even though is not really relevant. Does it make sense?